Library/Homebrew/test/sandbox_spec.rb

```diff
@@ -129,4 +129,22 @@ def initialize(*, **)
       end
     end
   end
+
+  describe "#disallow sending signal to other processes" do
+    # we have to spawn a process, otherwise kill doesn't try to send a signal if the process doesn't exist
+    let(:pid) do
+      pid = spawn("sleep 1000")
+      sleep 1 # Ensure the process has started
```

Comment:
Is this going to be an issue with slowing down the tests, or do we not care enough?

Reply:
```suggestion
      sleep 0.1 # Ensure the process has started
```